services/121-service/src/connection/connection.service.ts

```diff
@@ -465,6 +469,36 @@ export class ConnectionService {
     return note;
   }
 
+  public async updateAttribute(
```

Comment:
I have the feeling that there _must_ be some helper-methods/tools available in Nest.js to handle this "safely update an entity"-endpoint kind of method...

As it will be a very common thing in CRUD-APIs. I think this 'manual' approach will run 'out of hand' pretty quickly in this one method.

Reply:
Good point. I have just committed a new cleaner solution, which makes use of the fact that TypeOrm's .save() functionality also allows partial input and also automatically handles type conversion where necessary.

This new solution already handles customData attributes as well. So if you're on board with this, we could at least replace/remove the customDataOverwrite function with this one already. (Although we might need to check more things like phone number sanitization etc.)

I do realize this is scope creep and not actually part of this item..